include/openthread/link.h

```diff
@@ -1182,6 +1182,22 @@ bool otLinkIsCslSupported(otInstance *aInstance);
  */
 otError otLinkSendEmptyData(otInstance *aInstance);
 
+/**
+ * Sets the region code.
+ *
+ * The radio region format is the 2-bytes ascii representation of the ISO 3166 alpha-2 code.
+ *
+ * @param[in]  aInstance    The OpenThread instance structure.
+ * @param[in]  aRegionCode  The radio region code. The `aRegionCode >> 8` is first ascii char
+ *                          and the `aRegionCode & 0xff` is the second ascii char.
+ *
+ * @retval  OT_ERROR_FAILED           Other platform specific errors.
+ * @retval  OT_ERROR_NONE             Successfully set region code.
+ * @retval  OT_ERROR_NOT_IMPLEMENTED  The feature is not implemented.
+ *
+ */
+otError otLinkSetRegion(otInstance *aInstance, uint16_t aRegionCode);
```

Comment:
Should there be an associated `otLinkGetRegion()`?

Reply:
Added `otLinkGetRegion ()`.